### DISCUSSION TITLE
OpenVPN issues

### DISCUSSION THREAD

**Discussion by kiaraho** [Security-Onion-Solutions/securityonion#2444] [Unsupported Versions] [answered]
I ran into problems setting a ForwardNode over an OpenVPN connection, before I discovered issue #1139. I noticed the SO `2.3.21` setup scripts added the Management interface IP instead of the `tun0` interface IP to the firewall rules on the Manager. Manually adding the following firewall rules to the Manager was a messy, but somewhat functional workaround. I also had to manually restart the OpenVPN service on the sensor, after the SO setup scripts tore the `tun0` connection down.

```
sudo iptables -I INPUT 1 -p tcp --dport 4506 -s {sensor_tun0_ip} -d 0.0.0.0/0 -j ACCEPT
sudo iptables -I INPUT 2 -p tcp --dport 4505 -s {sensor_tun0_ip} -d 0.0.0.0/0 -j ACCEPT

sudo iptables -I DOCKER-USER 1 -p tcp --dport 5644 -s {sensor_tun0_ip} -d 0.0.0.0/0 -j ACCEPT
sudo iptables -I DOCKER-USER 2 -p tcp --dport 5044 -s {sensor_tun0_ip} -d 0.0.0.0/0 -j ACCEPT
sudo iptables -I DOCKER-USER 3 -p tcp --dport 443 -s {sensor_tun0_ip} -d 0.0.0.0/0 -j ACCEPT
sudo iptables -I DOCKER-USER 4 -p tcp --dport 8090 -s {sensor_tun0_ip} -d 0.0.0.0/0 -j ACCEPT
sudo iptables -I DOCKER-USER 5 -p tcp --dport 55000 -s {sensor_tun0_ip} -d 0.0.0.0/0 -j ACCEPT
sudo iptables -I DOCKER-USER 6 -p tcp --dport 8086 -s {sensor_tun0_ip} -d 0.0.0.0/0 -j ACCEPT
sudo iptables -I DOCKER-USER 7 -p tcp --dport 8080 -s {sensor_tun0_ip} -d 0.0.0.0/0 -j ACCEPT
sudo iptables -I DOCKER-USER 8 -p tcp --dport 5000 -s {sensor_tun0_ip} -d 0.0.0.0/0 -j ACCEPT
sudo iptables -I DOCKER-USER 9 -p tcp --dport 3142 -s {sensor_tun0_ip} -d 0.0.0.0/0 -j ACCEPT
```
Any word on when a proper fix for this issue will be released?

**Comment by weslambert** [comment, marked as answer]
This is being tracked through https://github.com/Security-Onion-Solutions/securityonion/issues/1139.

**Reply by kiaraho** [reply]
Thanks @weslambert. In the meantime, is there a temporary workaround, even if it’s unsupported?
